app/src/modules/collections/components/navigation-item.vue

```diff
@@ -6,7 +6,8 @@
 		:value="collection.collection"
 		query
 		:arrow-placement="collection.meta?.collapse === 'locked' ? false : 'after'"
-		@contextmenu="activateContextMenu"
+		@contextmenu.prevent.stop="activateContextMenu"
+		@focusout="deactivateContextMenu"
```

Comment:
I'm wondering if we can move this into a directive 🤔 The logic should be the same everywhere we use context menus

Reply:
Ya was thinking of simplifying it, but what I could thought of was only making them into composables 🙈 In any case, I wanted to put off the simplification and do that a separate PR, so I'll look into it since this is now merged.